Chatkit/Subscriptions/Instance/ResumableSubscription.swift

```diff
@@ -0,0 +1,9 @@
+import class PusherPlatform.PPResumableSubscription
+
+protocol ResumableSubscription: AnyObject {
+    var onOpen: (() -> Void)? { get set }
+    var onError: ((Error) -> Void)? { get set }
+    func end()
+}
+
+extension PusherPlatform.PPResumableSubscription: ResumableSubscription {}
```

Comment:
I don't believe it is exposed as public because `ResumableSubscription` is *not* itself public. 

Also its not actually possible to declare the extension as `internal` because it gives the following compilation error: 

`'internal' modifier cannot be used with extensions that declare protocol conformances`